server/config/users/userController.js

```diff
@@ -1,9 +1,14 @@
-
+var User = require('./userModel');
 
 exports.login = function(req, res) {
 
 };
 
 exports.signup = function(req, res) {
-
+  User.create({
```

Comment:
Shall we table the password encryption for a pre-release issue or just decide not to bother with it at all?